test/nn/conv/test_hypergraph_conv.py

```diff
@@ -46,3 +46,13 @@ def test_hypergraph_conv_with_more_edges_than_nodes():
     assert out.size() == (num_nodes, out_channels)
     out = conv(x, hyperedge_index, hyperedge_weight)
     assert out.size() == (num_nodes, out_channels)
+
+
+def test_hypergraph_jit():
+    in_channels, out_channels = (2, 3)
+    conv = HypergraphConv(in_channels, out_channels, use_attention=True)
+    script = torch.jit.script(conv)
+    output = script(torch.randn(4, in_channels),
+                    torch.tensor([[0, 1, 2], [0, 0, 1]]),
+                    hyperedge_attr=torch.randn(2, in_channels))
+    assert output.size() == (4, out_channels * conv.heads)
```

Comment:
```suggestion
    x = torch.randn(4, in_channels)
    out_1 = conv(x, torch.tensor([[0, 1, 2], [0, 0, 1]]),
                    hyperedge_attr=torch.randn(2, in_channels))
    out_2 = script(x,
                    torch.tensor([[0, 1, 2], [0, 0, 1]]),
                    hyperedge_attr=torch.randn(2, in_channels))
    assert torch.allclose(out_1, out_2)
```

Also add the `onlyFullTest` decorator like https://github.com/pyg-team/pytorch_geometric/blob/b2a1fdcf0d8f7622e00fec979b3e45eb384870c6/test/nn/pool/test_pool.py#L10. 
Good to merge after this.